salt/cloud/__init__.py

```diff
@@ -1261,7 +1261,12 @@ def create(self, vm_, local_master=True):
 
                 # a small pause makes the sync work reliably
                 time.sleep(3)
-                client = salt.client.get_local_client(mopts=self.opts)
+
+                mopts_ = salt.config.DEFAULT_MASTER_OPTS
+                mopts_.update(salt.config.minion_config('/etc/salt/master'))
```

Comment:
```python
os.path.join(opts['config_dir'] or syspaths.CONF_DIR, '<config-file-name>')
```

And just confirming, you're trying to load the minion configuration from the master configuration file?